metadata.rb

```diff
@@ -4,7 +4,7 @@
 license           'Apache 2.0'
 description       'Installs and configures haproxy'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '4.0.2'
+version           '4.0.3'
```

Comment:
@rshade and my bad on that I am still learning how to be a good contributor :) no problem! hahaha